tests/python/frontend/onnx/test_forward.py

```diff
@@ -30,6 +30,30 @@
 import scipy
 
 
+def get_tvm_output_with_vm(graph_def, input_data, target, ctx,
+                           opset=None, mode=None):
+    """ Generic function to execute and get tvm output with vm executor"""
+    if isinstance(input_data, list):
+        input_names = {}
+        shape_dict = {}
+        dtype_dict = {}
+        for i, _ in enumerate(input_data):
+            input_names[i] = graph_def.graph.input[i].name
+            shape_dict[input_names[i]] = input_data[i].shape
+            dtype_dict[input_names[i]] = input_data[i].dtype
+    else:
+        input_names = graph_def.graph.input[0].name
+        shape_dict = {input_names: input_data.shape}
+        dtype_dict = {input_names: input_data.dtype}
+
+    mod, params = relay.frontend.from_onnx(graph_def, shape_dict, opset=opset)
+
+    ex = relay.create_executor(mode, mod=mod, ctx=tvm.cpu(), target=target)
+    indata = tvm.nd.array(input_data)
+    result = ex.evaluate()(indata)
+    return result.asnumpy().transpose()
```

Comment:
Why transpose ?

Reply:
The output shape of argwhere is different from the one of ONNX nonzero. I move the transpose() to [here](https://github.com/apache/incubator-tvm/pull/5073/files#diff-03149f7671cff8be6734838f7707a24dR1455).